### ISSUE TITLE
The new "Rename Clip" command is poorly positioned in the Edit menu structure - would be better in Audio Clips submenu

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#2884]
**Describe the bug**
The new "Rename Clip" command is poorly positioned in the Edit menu structure, especially so since for 3.2.0 the old menu item for "Clip Boundaries" has been renamed to "Audio Clips".

With that change surely it would be better for the "Rename Clip" menu item to be in the submenu "Audio Clip" as that submenu deals with actions on Clips.

**To Reproduce**
1. Click on the Edit menu
2. Observe: " Rename Clip" is in this menu -see image 1
3. Click on "Audio Clips"
4. Observe:  all entries are actions on a Clip or Clips

**Actual behavior**
"Rename Clip" in in the top level Edit menu

**Expected behavior**
"Rename Clip"  to be in the submenu for "Audio Clips"

**Screenshots**
### 1) Edit Menu
![image](https://user-images.githubusercontent.com/50205984/166227217-b45826e9-9127-44b4-ad6b-a3812d2a08b8.png)

### 2) Audio Clips sub-menu
![image](https://user-images.githubusercontent.com/50205984/166227386-cb0c6f6d-2c77-4513-8d2f-e5b104049d28.png)

**Additional information (please complete the following information):**
 - OS: Windows 10 - but assume all OS
 - Version: 3.2.0

**Additional context**
I countered this as I have started work  _(planning only at present - the updating is tricksy)_  on updating the 3.2.0 alpha Manual for the recent changes to the Edit menu for this release.  The more I looked at it the more I thought that Rename Clip would be much better in the submenu for Audio Clips.

One also has to question just why this new menu item for Rename Clips has been added when there is a perfectly good mechanism in the right-click context menu for a click.

Tagging @Tantacrul and @LWinterberg as a design issue.

**Comment by petersampsonaudacity** [comment]
I will hold off working on updating this part of the Manual until this is resolved one way or the other, adopting or rejecting this proposal.

**Comment by LWinterberg** [comment]
Good idea. This only emerged because I renamed Clip Boundaries to Audio Clips :)

**Comment by DavidBailes** [comment]
When this command was introduced there was the intention at some point to generalize it so that it could be used for renaming either audio clips or labels. If that is still the intension, then that may well affect what is considered the best placement of the command in the menus.

**Comment by petersampsonaudacity** [comment]
> When this command was introduced there was the intention at some point to generalize it so that it could be used for renaming either audio clips or labels.

But we already have:  **Edit > Labels > **

**Comment by petersampsonaudacity** [comment]
Testing on W10 with @LWinterberg  latest branch   
audacity-win-3.2.0-alpha-20220502+4cb855d-x64-msvc2022 

The Rename Clip is NOT moved to the Audio Clips submenu - rather it's moved to the bottom of the top level Edit menu

![image](https://user-images.githubusercontent.com/50205984/166249952-2a2d7855-d453-4bfd-9657-1e1ffbc21f48.png)

**Comment by DavidBailes** [comment]
On Mon, 2 May 2022 at 13:52, Peter Sampson ***@***.***> wrote:

> When this command was introduced there was the intention at some point to
> generalize it so that it could be used for renaming either audio clips or
> labels.
>
> But we already have: **Edit > Labels > **
>
But I think the idea was to have a generic rename item command. @Paul
Licameli ***@***.***> may well have views on this.

**Comment by petersampsonaudacity** [comment]
BUMP

**Comment by petersampsonaudacity** [comment]
See also this related issue for the Clip context menu:
 **Clip context menu: the Rename Clip... would be better alongside the Split Clip #4235**

**Comment by petersampsonaudacity** [comment]
This is still the case in the latest 3.3.0 beta build

**Comment by petersampsonaudacity** [comment]
Still the case in  the recently released 3.3.1

**Comment by Tantacrul** [comment]
@petersampsonaudacity - unless we stated that we fixed an issue in 3.3.1 (but didn't), I don't think it's really needed to mention this. The issue isn't closed.

**Comment by chinakov** [comment]
This is fixed now, thank you!

**Comment by petersampsonaudacity** [comment]
I have updated the Manual for this change today.
